backend/Wheelingful.Data/Extensions/BuildModelExtension.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wheelingful.Data.Entities;
+
+namespace Wheelingful.Data.Extensions;
+
+internal static class BuildModelExtension
+{
+    public static void BuildModels(this ModelBuilder builder)
```

Comment:
do you need this method?

Reply:
It can be deleted, but it will appear again in the near future.